pubsub/subscriptions/subscription_test.go

```diff
@@ -144,13 +152,15 @@ func TestIAM(t *testing.T) {
 	})
 }
 
+// Not sure about the best way to rewrite this test
```

Comment:
Need to
1. Initialize a client.
1. Create a subscription if it doesn't already exist (can't depend on it already existing).
1. Call `delete`.
1. Check if the subscription still exists.